src/main/java/edu/kis/powp/jobs2d/command/ICompoundCommand.java

```diff
@@ -9,4 +10,5 @@
 
 	public Iterator<DriverCommand> iterator();
 
+	public void setCommands(List<DriverCommand> collect);
```

Comment:
IMPORTANT: I cannot agree to this modification. ICompoundCommand should not assume mutability!